func/db.sh

```diff
@@ -129,6 +133,125 @@ mysql_dump() {
 	fi
 }
 
+generate_policy_based_password() {
```

Comment:
While extremely creative I would recommend keeping things simple and replacing it with a long enough random generator:

```
tr -dc 'A-Za-z0-9%^&*()-_=+[]{};:,.<>?/' </dev/urandom | head -c 30;
```

I think it's better to keep passwords lengths long then just enough IMO. @jaapmarcus what do you think?

Reply:
Agree it is a lot of overhead for such simple step.